roles/ssh-manager/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+#SPDX-License-Identifier: MIT-0
```

Comment:
Only in one file 👀? Anyway, keep two things in mind:
- A single LICENSE file on the role directory also works. 
- There exists a [global license file](https://github.com/usegalaxy-eu/infrastructure-playbook?tab=MIT-1-ov-file) for the whole repository.